tasks/database_tests.py

```diff
@@ -231,7 +236,8 @@ def delete_accounts(context):
     #
     # Obtain our connection information and admin credentials
     #
-    couchdb_client_config = CouchDBClientConfig.load(couchdb_client_config_path=COUCHDB_CLIENT_CONFIG_PATH)
+    couchdb_client_config = CouchDBClientConfig.load(
```

Comment:
I've never used autopep8, and I kind of hate this code format (i.e., the way it handles parentheses).

But I'm not opposed to code formatting, might even think it's a good idea.

What do you think about Black?

https://black.readthedocs.io/en/stable/getting_started.html#do-you-like-the-black-code-style

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html